deploy/crds/app.terraform.io_workspaces_crd.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apiextensions.k8s.io/v1
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
`apiextensions.k8s.io/v1` would require k8s 1.16.0+. As a reference the current lowest version of EKS is 1.14 with an "end of support" date of December 8, 2020. Then 1.15 is supported until May, 2021. Might be best to stay on v1beta1 as it will be supported until 1.19.